Course_Part_06/src/Employee.java

```diff
@@ -0,0 +1,36 @@
+public class Employee {
+
+    int id;
+    String firstName;
+    String lastName;
+    int salary;
+
+    Employee(int id, String firstName, String lastName, int salary){
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.salary = salary;
+    }
+
+    public int getAnnualSalary() {
+        return this.salary * 12;
+    }
+
+    public void printAnnualSalary(){
+        System.out.println("Annual salary is: " + this.getAnnualSalary());
+    }
+
+    public void raiseSalary(int percent){
+        this.salary = this.salary * (percent+100) / 100;
+    }
+
+    public void printOverview(){
+        System.out.printf(
```

Comment:
Liever geen combinatie van String.format() en concatenation.

Liever deze implementatie:
`System.out.printf("Employee[id = %d, name = %s %s, salary = %d]%n",
                id,
                firstName,
                lastName,
                salary
                );`

Reply:
Uitlijning gaat niet lekker in Github, excuus